templates/patterns/featured_media/pattern-featured-media.html.twig

```diff
@@ -24,7 +24,14 @@
       {% set video_alignment_class = 'order-md-1' %}
     {% endif %}
 
-    <h2 class="mb-4">{{ title }}</h2>
+    {%- if title is not empty -%}
+      {% include '@oe-bcl/heading' with {
+        title: title,
+        title_tag: title_tag|default('h2'),
```

Comment:
the type of the `title` field in `featured_media.ui_patterns.yml` is text, there is no title_tag nor title_link

Reply:
I added also title_tag and title_link in yml file.